loader.php

```diff
@@ -24,6 +24,17 @@ function atomic_blocks_loader() {
 	 */
 	require_once plugin_dir_path( __FILE__ ) . 'dist/getting-started/getting-started.php';
 
+	/**
+	 * Load Migrate page
+	 */
+	require_once plugin_dir_path( __FILE__ ) . 'dist/migration/migrate-page/migrate-page.php';
+
+	/**
+	 * Enable API Endpoint for installing Genesis Blocks.
+	 */
+	require_once plugin_dir_path( __FILE__ ) . 'dist/migration/class-install-genesis-blocks-api.php';
+	new Atomic_Blocks\Admin\Migration\Install_Genesis_Blocks_Api();
```

Comment:
Related to other comment, recommend changing `Atomic_Blocks` to `AtomicBlocks` for consistency with rest of plugin.